pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Enterprise apps on Amazon EKS. In about a day.
+excerpt: Build battle-tested Amazon EKS clusters, designed for enterprise with security, observability, and usability built-in. Manage 100% of your infrastructure using code, all in ~1 day. <b>Join now for special early access pricing!</b>
+permalink: /eks-for-enterprise/
+custom_js:
+- prism
+prism_css: /assets/css/prism-landing-zone.css
+
+technologies:
+- name: Spring Boot
+  logo: /assets/img/logos-technologies/logo-spring-boot.png
+- name: Ruby on Rails
+  logo: /assets/img/logos-technologies/logo-rails.png
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Request early access
+  logos:
+  - /assets/img/logos-technologies/partner_logo_aws.png
+  - /assets/img/logos-technologies/partner_logo_kubernetes.png
+  - /assets/img/logos-technologies/partner_logo_terraform.png
+
+how_it_works:
+  - heading: "Step 1: Choose your EKS configuration"
+    precode: |
+      <figure><figcaption>eks-configuration.yml</figcaption>
+        <pre>
+            <code class="language-yml">
+            # Configure CIS Benchmark compliance
+            cis:
+              enabled: true
+
+            # Choose from istio, consul-connect, aws-app-mesh
+            service_mesh: "istio"
+
+            # Choose from flux, jenkins, argocd
+            cicd: "flux"
+
+            # Choose from prometheus, datadog, cloudwatch
+            observability: prometheus
+
+            # Configure various common utilities
+            cluster_autoscaler:
+              enabled: true
+
+            sealed_secrets:
+              enabled: true
+
+            aws_load_balancer_controller:
+              enabled: true
+
+            external_dns:
+              enabled: true
+              domain: gruntwork.io
+            </code>
+        </pre>
+      </figure>
+    image_side: right
+    body: |
+      Pick and choose the technologies that you want included in your custom EKS cluster. We
+      support a wide range of popular configurations that include:
+      <ul>
+          <li>Service meshes (Istio, Consul connect, AWS App Mesh)</li>
+          <li>GitOps CI/CD (Flux, Jenkins, Argo)</li>
+          <li>UI dashboards (Porter, Grafana)</li>
+          <li>Observability (Prometheus, Datadog, CloudWatch, ELK)</li>
+          <li>And more (cluster-autoscaler, sealed secrets, AWS Load Balancer Controller, etc)</li>
+      </ul>
+      On top of that, all our enterprise-grade EKS clusters are hardened according to the <a
+      href="https://aws.amazon.com/blogs/containers/introducing-cis-amazon-eks-benchmark/">CIS Amazon EKS
+      Benchmark</a>, regardless of the chosen technologies.
+    logos:
+      - title: CIS
+        src: /assets/img/logos-compliance/cis-benchmark.png
+      - title: EKS
+        src: /assets/img/logos-technologies/logo-eks.png
+
+  - heading: "Step 2: Deploy your EKS cluster"
```

Comment:
It's not super clear how you go from step 1 to step 2. What happens in between? Are we opening a PR in the customer's repo with generated code? The description below talks about generating code for accounts and VPCs, but it's not clear how the EKS cluster itself is managed. And how do you deploy it? Does our UI do it? Or some CI/CD pipeline? Or are you expected to run `apply` yourself?

Note that the fact that the config for the entire EKS cluster is managed as code is a [key differentiator](https://www.notion.so/gruntwork/Enterprise-Enterprise-EKS-344f903153b448d2a93fd9edc82adb42#677be35f65f44e5c86ca1214ed252248) for us from all the other Enterprise K8S platforms, so it's important to get it across here. 

I don't know the best solution here, but some thoughts:

- Perhaps an extra step in between showing that we open a PR? And explicitly call out that (a) we generate the code into a PR, (b) the code is based on Gruntwork's prod-grade IaC Lib that has been used by hundreds of companies, and (c) you can customize this code any way you like.
- If we do have the PR as a separate step, then the deploy step can focus explicitly on running `apply` to deploy the thing. We can mention first-class support for using Terraform, Terraform Cloud, Terraform Enterprise, or Terragrunt.
- The deploy step could also mention that we can optionally set up an infra CI / CD pipeline for you... But that might be too much, and confuse things.

Reply:
I came here specifically to +1 this. I'm a more savvy consumer of dev tools now, but I remember when I was first learning DevOps, I'd see tools like https://web.archive.org/web/20200311034438if_/https://containership.io/ and not really understand what it did:

"Ok, it helps launch K8s...as a platform...but how does that relate to EKS...and I'm using Terraform...so?"

I worry our current page might suffer from the same issue where users won't really get our unique angle. As Jim said, our unique angle is that _we give you a bunch of Terraform code._ We mention that in the page, but we don't make that a top-level positioning.

One suggestion might be to update our hero text to something like:

> Amazon EKS for the enterprise. 
> Delivered as Terraform code.
> Deployed in about a day.

Reply:
See https://gruntwork-io.slack.com/archives/C66H4D4TZ/p1628124336004300 for another example of "I don't get what problem this dev tool solves."

Reply:
I added in a generate code step and modified the deploy section so it's a bit clearer.

Reply:
Also updated the hero text taking Josh's suggestion as I think that does make more sense here.